app/qml/project/MMProjectController.qml

```diff
@@ -200,7 +200,7 @@ Item {
           return qsTr("Home")
         }
         else if ( projectsPage.state === "workspace" ) {
-          return qsTr("Projects")
+          return __merginApi.userInfo.activeWorkspaceName
```

Comment:
In case you do not have any active workspace 

```suggestion
          return __merginApi.userInfo.activeWorkspaceName ? __merginApi.userInfo.activeWorkspaceName : qsTr( "Projects" )
```